ACube.js/html/inc/gacube2.js

```diff
@@ -0,0 +1,313 @@
+const PYRA_SVG = `
```

Comment:
ideally these would be included elsewhere and have some sort of radio/dropdown input to select them